pingcap/tidb
# *: make QPS more realistic

Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>



### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:
The QPS pannel is exactly CPS. Try to fix it.
<img width="1362" alt="企业微信截图_20200602131855" src="https://user-images.githubusercontent.com/4352397/83486561-02812580-a4dc-11ea-86e5-e2652552ca54.png">

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:
<img width="1515" alt="企业微信截图_20200602132556" src="https://user-images.githubusercontent.com/4352397/83486573-08770680-a4dc-11ea-8221-affc245fdd34.png">

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
1. start a cluster
1. run sysbench update non index
1. check out the monitor

### Release note 
- No release note.
